app/controllers/content_items_controller.rb

```diff
@@ -20,14 +20,21 @@ def update
       ContentItem.create_or_replace(encoded_base_path, @request_data)
     end
 
-    if result
-      status = (result == :created ? :created : :ok)
-    else
+    response_body = {}
+    case result
+    when :created
+      status = :created
+    when :stale
+      status = :conflict
+      response_body = { errors: "A later transmitted_at is already stored" }
```

Comment:
Also, this message will be returned if `transmitted_at` isn't implemented, discarding the error message from `UpdateLock`.